### DISCUSSION TITLE
After adding "implements HasMedia" phpStorm complains

### DISCUSSION THREAD

**Discussion by pathros** [spatie/laravel-medialibrary#2358] [Q&A] [answered]
When I add 
`implements HasMedia`
to my Model, phpStorm complains:

_Declaration of 'InteractsWithMedia->clearMediaCollection([collectionName: string = 'default'])' must be compatible with 'HasMedia->clearMediaCollection([collectionName: string = 'default'])'_ 

This doesn't seem to affect the MediaLibrary functionality. Just to hear if somebody has a fix or a workaround ... or if it's some bug of phpStorm

**Comment by freekmurze** [comment, marked as answer]
It's a PhpStorm bug. It's on their tracker, hope they'll resolve it soon.
